org.bino3d.bino.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.bino3d.bino
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: bino
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Play videos from the net
+  - --share=network
+  # OpenGL rendering
+  - --device=dri 
+  # Webcam capturing
+  - --device=all
+  # Audio output
+  - --socket=pulseaudio
+  # Play video files, but only read access required
+  - --filesystem=host:ro
```

Comment:
how about `xdg-videos` instead?

Reply:
That would probably be fine in GUI mode, but in Virtual Reality mode for large-scale systems with a render cluster, there is typically a dedicated cluster filesystem involved that users are encouraged to use for data that cluster nodes need to access. Those use cases are not typically covered by the xdg specs, but it's one of the focus areas of Bino.